src/ion/blueprints/utils.rs

```diff
@@ -112,6 +112,110 @@ pub fn list_templates(config: &Config) -> Result<()> {
     Ok(())
 }
 
+pub fn inspect_template(config: &Config, template_name: String) -> Result<()> {
+    let templates = config.template_dir().read_dir()?;
+
+    let mut template_found: bool = false;
+
+    for entry in templates {
+        let entry = match entry {
+            Ok(e) => e,
+            Err(e) => {
+                return Err(Error::new(e));
+            }
+        };
+
+        let path = entry.path();
+        if path.is_dir() {
+            let source = std::fs::read_to_string(path.join("template.toml"))?;
+
+            let template = match toml::from_str::<Template>(&source) {
+                Ok(t) => t,
+                Err(e) => {
+                    return Err(format_err!("Error parsing template: {}", e));
+                }
+            };
+            if template.name == template_name {
+                template_found = true;
+                println!("{}", source);
+            }
+        }
+    }
+
+    // If the template the user requested is not in the list of downloaded templates, ask user to select existing template to inspect
+    if !template_found {
+        println!(
+            "The {} template was not found.\nInstalled templates are:",
+            template_name
+        );
+        ask_inspect_template(config)?
+    }
+    Ok(())
+}
+
+pub fn inspect_all_templates(config: &Config) -> Result<()> {
+    let templates = config.template_dir().read_dir()?;
+
+    for entry in templates {
+        let entry = match entry {
+            Ok(e) => e,
+            Err(e) => {
+                return Err(Error::new(e));
+            }
+        };
+
+        let path = entry.path();
+        if path.is_dir() {
+            let source = std::fs::read_to_string(path.join("template.toml"))?;
```

Comment:
we probably should just provide a pretty-printing for `Template` so we can just use that here. But I think directly print the source is also good, we can improve it later.